docs/output.md

```diff
@@ -1,35 +1,105 @@
-# nf-core/mag
-Assembly, binning and annotation of metagenomes
+# nf-core/mag: Output
 
 This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
 
 ## Pipeline overview
+
 The pipeline is built using [Nextflow](https://www.nextflow.io/)
 and processes data using the following steps:
 
-* [FastQC](#fastqc) - read quality control
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+- [FastQC](#fastqc) - read quality control
+- [fastp](#fastp) - read trimming
+- [megahit](#megahit) - assembly
+- [quast](#quast) - assembly quality report
+- [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+- [metabat](#metabat) - recovered draft genome bins from assembled metagenomes
+- [checkm](#checkm) - genome bins quality control and merging of compatible bins
+- [refinem](#refinem) - improving metagenome-assembled genome bins
 
 ## FastQC
+
 [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
 
 For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
 
+> **NB:** The FastQC plots displayed in the MultiQC report shows both _untrimmed_ and _trimmed_ reads.
+
 **Output directory: `results/fastqc`**
 
-* `sample_fastqc.html`
-  * FastQC report, containing quality metrics for your untrimmed raw fastq files
-* `zips/sample_fastqc.zip`
-  * zip file containing the FastQC report, tab-delimited data file and plot images
+- `sample_fastqc.html`
+  - FastQC report, containing quality metrics for your untrimmed raw fastq files
+- `zips/sample_fastqc.zip`
+  - zip file containing the FastQC report, tab-delimited data file and plot images
+
+## fastp
+
+[fastp](https://github.com/OpenGene/fastp) is a all-in-one fastq preprocessor for read/adapter trimming and quality control. It is used in this pipeline for trimming adapter sequences and discard low-quality reads.
+
+**Output directory: `None`**
+
+- The trimmed reads are not included in the output
```

Comment:
It can be nice to make it an option to save intermediate files such as these. In the RNAseq pipeline we use `--saveTrimmed` to keep them for example:

https://github.com/nf-core/rnaseq/blob/e8376373da8c1dfb0cae5372947cf5b241056076/main.nf#L474-L480